application/common/config/main.php

```diff
@@ -32,18 +32,14 @@
     $emailServiceConfig['validIpRanges'] = '';
 }
 
-if (empty($hrNotifierEmailTo)) {
```

Comment:
What has changed that let's this code be removed?

Reply:
Because of the checks in `sendMissingEmailNotice` and `getEmailTo`

Reply:
Okay.